lalrpop-internals/src/parser/lrgrammar.rs

```diff
@@ -1,4 +1,4 @@
-// auto-generated: "lalrpop 0.22.1"
+// auto-generated: "lalrpop-internals 1.0.0"
```

Comment:
Just noticed this change.  I feel like we probably want to keep "lalrpop 0.22.1" here.  I'll investigate.

Reply:
Hmm, that seems a little challenging because you are looking for the version of a user of this crate. In some sense, the most precise/accurate versioning is of the `lalrpop-internal` version.

👍 Interested to see what you find